shell/auto_test/tests/org/organization-settings.spec.ts

```diff
@@ -0,0 +1,148 @@
+    await page.click('text=joint issue type');
+    await page.click('text=requirement');
+    await page.click('input[role="combobox"]');
+    await page.click(':nth-match(:text("test"), 2)');
+    await page.click('button:has-text("reference")');
+    await page.click('text=testTextremovemove upmove down >> span');
+    await page.click('button:has-text("OK")');
+    await page.click('li:has-text("block network")');
+    expect(page.url()).toMatch(/\/orgCenter\/setting\/detail\?tabKey=block-network/);
+    await page.click('text=prod environmentoff >> button[role="switch"]');
+    await page.click('button:has-text("OK")');
+    await expectExist('text=on');
+    await page.click('button[role="switch"]:has-text("on")');
+    await page.click('button:has-text("OK")');
+    await expectExist('text=off');
+    await page.click('#main >> text=audit log');
+    expect(page.url()).toMatch(/\/orgCenter\/setting\/detail\?tabKey=operation%20log/);
+    await page.click('[aria-label="Increase Value"]');
+    await page.click('button:has-text("update")');
+    expectExist('text=updated successfully', 0);
+    await page.click('li:has-text("notification group")');
+    expect(page.url()).toMatch(/\/orgCenter\/setting\/detail\?tabKey=notifyGroup/);
+    await page.click('button:has-text("new notification group")');
+    await page.click('input[type="text"]');
+    await page.fill('input[type="text"]', testData.title);
+    await page.click('text=group namenotified to >> :nth-match(span, 2)');
+    await page.click('div[role="document"] >> text=member');
+    await page.click('.ant-dropdown-trigger');
+    await page.click('[placeholder="search by keywords"]');
+    await page.fill('[placeholder="search by keywords"]', testData.memberName);
+    await page.click(`text=${testData.memberName}(${testData.manager})`);
+    await page.click('text=new Group');
+    await page.click('button:has-text("ok")');
+    await expectExist(`text=${testData.memberName}`);
+    await page.click('text=org info');
+    expect(page.url()).toMatch(/\/orgCenter\/setting\/detail\?tabKey=orgInfo/);
+    await expectExist('text=organization identifier');
+    await expectExist('text=org name');
+    await expectExist('text=notice language');
+    await expectExist('text=org description');
+    await page.click('button:has-text("edit")');
+    await expectExist('text=edit org info', 1);
+    await expectExist('.ant-input-disabled', 1);
+    await page.click('#displayName');
+    await page.fill('#displayName', testData.title);
+    await page.click('span:has-text("Chinese")');
+    await page.click('text=English');
+    await page.click('text=English');
+    await page.click('div[role="document"] >> text=Chinese');
+    await page.click('text=private org');
+    await page.click('span:has-text("public org")');
+    await page.click(
+      'svg:has-text("@font-face{font-family:feedback-iconfont;src:url(//at.alicdn.com/t/font_1031158_")',
+    );
```

Comment:
url is mutable，don‘t use unstable selector

Reply:
ok,modified